chromium_src/chrome/browser/ui/views/tabs/tab_container.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/tabs/brave_tab_container.h"
+#include "brave/browser/ui/views/tabs/features.h"
+
+#include "src/chrome/browser/ui/views/tabs/tab_container.cc"
+
+// In order to instantiate RemoveTabDelegate which is defined in
+// tab_container.cc, this method is defined here.
+void BraveTabContainer::StartRemoveTabAnimation(Tab* tab,
```

Comment:
same as BraveTabDragContext. We absolutely cannot have copied code from upstream here. This cannot break or cause any problems with regular tabs and copying code will almost certainly result in that happening over time.